processes/processes.tex

```diff
@@ -274,22 +311,24 @@ \subsection{Fork Functionality}
   }
 \end{lstlisting}
 
-The algorithm isn't actually O(N) because of how the system scheduler works.
+The algorithm isn't actually O(N) because of how the system scheduler works. In essence, this program outsources the actual sorting to the operating system (particularly the scheduler).
 
 \subsection{What is a fork bomb?}
 
-A fork bomb is what we warned you about earlier.
-A `fork bomb' is when you attempt to create an infinite number of processes.
-This will often bring a system to a near-standstill as it attempts to allocate CPU time and memory to a very large number of processes that are ready to run.
-System administrators don't like fork-bombs and may set upper limits on the number of processes each user can have or may revoke login rights because it creates a disturbance in the force for other users' programs.
-You can also limit the number of child processes created by using \keyword{setrlimit()}. fork bombs are not necessarily malicious - they occasionally occur due to coding errors.
+A `fork bomb` is what we warned you about earlier.
```

Comment:
We should use the latex start quote ` and the end quote '

Reply:
Done.